app/data/tasks/export_csv.py

```diff
@@ -55,9 +57,10 @@ def export_csv(query_key):
     # Add a directory for the schema we're outputting
     dirname = 'schema-' + str(schema.pk) + '/'
     for f, name in record_writer.get_files_and_names():
-        info = zipfile.ZipInfo(dirname + name)
+        info = zipfile.ZipInfo(filename=dirname + name, date_time=time.localtime(time.time())[:6])
         info.external_attr = 0755 << 16L
-        archive.writestr(info, f.read())
+        z = open(f.name, 'r')
```

Comment:
This would be better as a context manager so that we don't have to worry about closing the file afterward, i.e.
```python
with open(f.name, 'r') as z:
    archive.writestr(info, z.read())
```